spec/models/miq_expression_spec.rb

```diff
@@ -321,19 +321,19 @@
       it "generates the SQL for an IS expression with a 'Today' value for a date field" do
         exp = MiqExpression.new("IS" => {"field" => "Vm-retires_on", "value" => "Today"})
         sql, * = exp.to_sql
-        expect(sql).to eq("vms.retires_on BETWEEN '2011-01-11' AND '2011-01-11'")
+        expect(sql).to eq("\"vms\".\"retires_on\" BETWEEN '2011-01-11' AND '2011-01-11'")
       end
 
       it "generates the SQL for an IS expression with an 'n Hours Ago' value for a date field" do
         exp = MiqExpression.new("IS" => {"field" => "Vm-retires_on", "value" => "3 Hours Ago"})
         sql, * = exp.to_sql
-        expect(sql).to eq("vms.retires_on BETWEEN '2011-01-11' AND '2011-01-11'")
+        expect(sql).to eq("\"vms\".\"retires_on\" BETWEEN '2011-01-11' AND '2011-01-11'")
       end
 
       it "generates the SQL for an IS expression with an 'n Hours Ago' value for a datetime field" do
         exp = MiqExpression.new("IS" => {"field" => "Vm-last_scan_on", "value" => "3 Hours Ago"})
         sql, * = exp.to_sql
-        expect(sql).to eq("vms.last_scan_on BETWEEN '2011-01-11T14:00:00Z' AND '2011-01-11T14:59:59Z'")
+        expect(sql).to eq("\"vms\".\"last_scan_on\" BETWEEN '2011-01-11 14:00:00' AND '2011-01-11 14:59:59'")
```

Comment:
@carbonin, @imtayadeway noticed that when the SQL for this expression is built by Arel, it removes the timezone designation from it ('2011-01-11T14:00:00Z' vs. '2011-01-11 14:00:00'). We're not sure if we should be concerned about that or not. It seems that when the query is executed that Postgres will compare the date/time as is with what's in the table without doing any conversion. And, since all of our timstamps in the DB are in UTC and our application converts timestamps to UTC before building the queries, we should be good. What do you think?